packages/perseus/src/components/math-input/math-input.tsx

```diff
@@ -13,7 +11,7 @@ import {
 import Clickable from "@khanacademy/wonder-blocks-clickable";
 import {View} from "@khanacademy/wonder-blocks-core";
 import {Popover, PopoverContentCore} from "@khanacademy/wonder-blocks-popover";
-import {color, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {color, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
 import {HeadingMedium} from "@khanacademy/wonder-blocks-typography";
 import {StyleSheet} from "aphrodite";
```

Comment:
(comment on file/dir, not this line of code)

I don't think we need to add a `math-input` folder if the only file in it is just `math-input.tsx`. If for some reason we do need the `math-input` folder maybe it would make sense if this file was `index.tsx` so that we don't need to update imports.